syntax = "proto3";
package order;

import "google/protobuf/timestamp.proto";


message OrderRequestPB {
  string customer_email = 1;
  int32 offer_id = 2;
  int32 number_of_adults = 3;
  int32 number_of_children = 4;
  float price = 5;
}

message OrderPB {
  int32 id = 1;
  int32 offer_id = 2;
  float price = 3;
  int32 number_of_adults = 4;
  int32 number_of_children = 5;
  google.protobuf.Timestamp creation_time = 6;
}


message OrderListRequestPB {
  string customer_email = 1;
}

message OrderListPB {
  repeated OrderPB orders = 1;
}
